generators/generator-base.js

```diff
@@ -262,29 +270,41 @@ Generator.prototype.getAllInstalledLanguages = function () {
  * get all the languages supported by JHipster
  */
```

Comment:
We should be careful when changing method signatures of these methods as it could be used by modules. We should change only if its absolutely necessary